ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -1203,7 +1337,8 @@ def patch():
         wrap("langchain", "embeddings.OpenAIEmbeddings.embed_documents", traced_embedding(langchain))
     else:
         from langchain.chains.base import Chain  # noqa:F401
-        from langchain_core.tools import BaseTool  # noqa:F401
```

Comment:
Removing this didn't break patching?

Reply:
ah i didn't mean to remove this line, thanks. but, it looks like all tests still passed. i'll put it back since it's out of scope for this PR, but maybe we can try removing it in a different one if we don't need it here